### ISSUE TITLE
Irritating error message after deployment of ejb based webservice in war

### ISSUE THREAD

**Issue by matthiasblaesing** [payara/Payara#384]
We encountered the following issue when deploying an EJB-based webservice that is packed as a war:

```
[2015-08-06T11:55:20.358+0200] [Payara 4.1] [SEVERE] [] [global] [tid: _ThreadID=47 ThreadName=admin-listener(5)] [timeMillis: 1438854920358] [levelValue: 1000] [[
  Component referenced from annotation symbol cannot be found
 symbol: javax.jws.HandlerChain
 location: class blaesing.demoproject.TestService]
```
This message is problematic because my collegue was tracking down a problem where the handlerchain file was not found and when he got to a state where he was faced with the above error message (with his correct Handler of course). He thought now he had another problem.

Sources can be found here: https://github.com/matthiasblaesing/PayaraHandlerChainWarning

That assumption proved wrong, as the handler is in fact invoked for the webservice:

```
[2015-08-06T11:55:20.979+0200] [Payara 4.1] [INFO] [] [javax.enterprise.system.core] [tid: _ThreadID=47 _ThreadName=admin-listener(5)] [timeMillis: 1438854920979] [levelValue: 800] [[
  PayaraHandlerChainWarning was successfully deployed in 642 milliseconds.]]

[2015-08-06T11:55:33.554+0200] [Payara 4.1] [INFO] [] [] [tid: _ThreadID=27 _ThreadName=Thread-8] [timeMillis: 1438854933554] [levelValue: 800] [[
  Handler called]]

[2015-08-06T11:55:33.564+0200] [Payara 4.1] [INFO] [] [] [tid: _ThreadID=27 _ThreadName=Thread-8] [timeMillis: 1438854933564] [levelValue: 800] [[
  Handler called]]
```

So the the question issue is: Either there is an error and we don't see it or the message is wrong either way it looks like an issue in glassfish/payara.

The test was done on payara 4.1.152.

If you need further information, feel free to request it.

**Comment by smillidge** [comment]
Thanks for the information and the test case we will try to reproduce

**Comment by smillidge** [comment]
Reproduced on current master

**Comment by lprimak** [comment]
Should be fixed with the above commit

**Comment by smillidge** [comment]
Fixed by #812
